lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskRuntime.cs

```diff
@@ -113,14 +113,17 @@ public Thread StartTaskOnNewThread()
                 }
                 catch (TaskStartHandlerException e)
                 {
+                    Logger.Log(Level.Info, "TaskRuntime::TaskStartHandlerException");
```

Comment:
These changes are orthogonal to the changes for fault tolerance. Maybe it would be a good idea to factor them (and any other straight-forward quality improvements) into their own PR?